ff7-asset-loader/kernel-section-init-data.js

```diff
@@ -223,13 +227,13 @@ const getInitSectionData = (sectionData, itemNames, itemDescriptions, materiaNam
         portrait1: 0,
         portrait2: 0,
         portrait3: 0,
-        name: '',
+        leader: '',
         currentHP: 0,
         maximumHP: 0,
         currentMP: 0,
         maximumMP: 0,
         gil: 0,
-        seconds: 0,
+        time: 0,
```

Comment:
BTW, I have this habit of sometimes making my variable names long and descriptive. For example, 5 years ago, I would have probably named this variable "timeSeconds".

It stems from working at Google 12 years ago, on a Java code base with lots of contributors and a huge risk of impacting revenue if mistakes were made. Variables like "time" with pure numerical values were banned and rejected, because they are ambiguous; you won't know whether the value is supposed to be in milliseconds or seconds or whatever.

The downside with long var names is that they take longer to type and longer to read. It prevents devs from understanding the code at a glance, and slows down development in general. So it's actually NOT a good style for many modern projects.

Obviously, it all depends on the context.

So over the last few years, I've had to un-train myself and allow myself to use shorter var names. At one point I used single-letter var names a lot, but those were too short, I couldn't understand it a week later.

Nevertheless, 1- or 2- letter var names can still be appropriate in certain areas. Like, assembly language! Or, old code that was written at a time when disk space was a premium. (And both of these situations apply to FF7 op codes!)

This situation is weird because I wanted kujata to make FF7 op codes (and all other data in general) to be more accessible and understandable, which lends towards longer names. Then again, if you're a developer, and if you have spent a lot of time on them, then you get to a point where you prefer super short names. Like "t" for time.

Side note: If we had an external "schema", we could provide metadata for each var, and possibly even define both "long" and "short" names. But that's a whole new endeavor all by itself!

Anyway... oh, btw, for this line of code, "time" is great here. No action for you here. I'm just rambling.

Actually, later on, if you wanted to change **all** the var names to be shorter (e.g. "t"), I might even be okay with that. Although that might be a lot of refactoring for you. :)

And in general, if there are any code style changes or other kinds of changes that would make life easier for you to work on your engine / projects, just let me know! You're the main "customer" of this code base, Dan!